### DISCUSSION TITLE
How can I make my mesh appear solid from all camera angles?

### DISCUSSION THREAD

**Discussion by pvasired** [isl-org/Open3D#5226] [Q&A]
I am trying to convert a point cloud into a mesh, but I am having trouble getting the orientation of my surface normals correct for what I want.

I want the final shape to look solid from all camera angles rather than becoming transparent as I rotate my camera angle around. An example of what I mean is below.

![badmesh](https://user-images.githubusercontent.com/17258295/174680381-67773326-1d8b-4e33-863b-749afca51111.png)

A snippet of my code is as follows:

```
pcd = o3d.geometry.PointCloud()
pcd.points = o3d.utility.Vector3dVector(points)
pcd.estimate_normals()
pcd.orient_normals_towards_camera_location()
mesh, _ = o3d.geometry.TriangleMesh.create_from_point_cloud_poisson(pcd, depth=3)
mesh.compute_vertex_normals()
mesh.paint_uniform_color(np.array([0,191,255])/255)
```

I would greatly appreciate any help on this!

**Comment by AshengLin** [comment]
Maybe you can use open3d.visualization.draw_geometries([mesh], mesh_show_back_face=True)?
